### PR TITLE
feat: add and configure Serverless plugins to app

### PR DESCRIPTION
**Pull request by svetlanagorchanina**
Serverless-finch and serverless-single-page-app plugins are added and configured.

Link to CloudFront website: https://d2fq48c0jcr952.cloudfront.net
S3 link: http://nodejs-aws-app-2.s3-website-us-east-1.amazonaws.com/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,9 @@ yarn-error.log*
 
 /.idea
 .vscode
+
+# package directories
+jspm_packages
+
+# Serverless directories
+.serverless
```
Comment (NikaOrl) [inline .gitignore]: .serverless already exists in .gitignore

File: package.json [hunk 2]
```diff
@@ -6,33 +6,49 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "deploy:s3": "serverless client deploy --no-config-change --no-policy-change --no-cors-change",
```
Comment (NikaOrl) [inline package.json]: duplicates 'client:deploy' script

File: package.json [hunk 3]
```diff
@@ -6,33 +6,49 @@
...
+    "build:deploy": "npm run build && npm run deploy:s3",
```
Comment (NikaOrl) [inline package.json]: duplicates 'client:build:deploy' script

**Review by NikaOrl — CHANGES_REQUESTED** [review]
Please, follow convention for commit messages: https://docs.rs.school/#/git-convention and convention for PR description: https://docs.rs.school/#/stage2?id=Требования-к-pull-request-pr

**Comment by NikaOrl** [comment]
Score: 5/5

**Comment by svetlanagorchanina** [comment]
@NikaOrl Good points, thanks for great review!

**Pull request merged**